examples/pipelines/datacomp/components/filter_text_complexity/requirements.txt

```diff
@@ -1,4 +1,4 @@
-git+https://github.com/ml6team/fondant.git@main
+git+https://github.com/ml6team/fondant.git@6ef449cabcd56a59b2d4405e82060b745f992560
```

Comment:
This was only needed for you locally to invalidate the cache. We don't need to add this to git. I'll open a PR soon that tries to tackle these issues.